### PR TITLE
Add new volunteer Sudha

### PR DESCRIPTION
**Pull request by surani-hub**

### CODE REVIEW

File: data/users/surani-hub.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Sudharani",
+  "bio": "Java Developer",
+  "image": "https://avatars0.githubusercontent.com/u/7663490?s=400&v=4",
+  "categories": [
+    {
+      "slug": "programming | software-development",
```
Comment (RakeshSangem) [inline data/users/surani-hub.json:7]: Hey @surani-hub thanks for volunteering.

You need to choose only one slug. You can find these in `public/categories.json`

**Review by RakeshSangem — CHANGES_REQUESTED** [review]

File: data/users/surani-hub.json (hunk 1 above)
Comment (surani-hub) [reply data/users/surani-hub.json]: Hi @RakeshSangem, I just updated the slug to one field. Could you please check and verify?

Comment (codewithshubhi) [reply data/users/surani-hub.json]: Hey @surani-hub this should work thanks :)

File: data/users/surani-hub.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Sudharani",
+  "bio": "Java Developer",
+  "image": "https://avatars0.githubusercontent.com/u/7663490?s=400&v=4",
+  "categories": [
+    {
+      "slug": "programming",
+      "topics": ["Java", "Data structures", "Algorithms", "MySQL"]
```
Comment (codewithvoid) [inline data/users/surani-hub.json:8]: Keep this to max 2-3.

Also MySQL requires another category `Database Development`

Comment (surani-hub) [reply data/users/surani-hub.json:8]: Hi @codewithvoid, would you want me to keep the topics to just 3 and remove the remaining?

Comment (RakeshSangem) [reply data/users/surani-hub.json:8]: > Hi @codewithvoid, would you want me to keep the topics to just 3 and remove the remaining?

yes!

**Comment by codewithvoid** [comment]
Closing this as it's duplicate of #75

**Pull request closed without merging**